docs/conf.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python
+#
+# xscen documentation build configuration file, created by
+# sphinx-quickstart on Fri Jun  9 13:47:02 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another
+# directory, add these directories to sys.path here. If the directory is
+# relative to the documentation root, use os.path.abspath to make it
+# absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+import xscen  # noqa
+import xarray  # noqa
+
+xarray.DataArray.__module__ = "xarray"
+xarray.Dataset.__module__ = "xarray"
+
+# -- General configuration ---------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.viewcode",
+    "sphinx.ext.mathjax",
+    "sphinx.ext.napoleon",
+    "sphinx.ext.coverage",
+    "sphinx.ext.todo",
+    "sphinx.ext.autosectionlabel",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.extlinks",
+    "nbsphinx",
+]
+
+# To ensure that underlined fields (e.g. `_field`) are shown in the docs.
+autodoc_default_options = {
+    "members": True,
+    "undoc-members": True,
+    "private-members": False,
+    "special-members": False,
+}
+
+autosummary_generate = True
+nbsphinx_execute = "auto"
+
+# To avoid having to install these and burst memory limit on ReadTheDocs.
+# autodoc_mock_imports = [
+#     "cartopy",
+#     "clisops",
+#     "dask",
+#     "h5py",
+#     "intake",
+#     "intake_esm",
+#     "pandas",
+#     "rechunker"
+#     "xarray",
+#     "xclim",
+#     "xesmf",
+#     "zarr",
+# ]
```

Comment:
```suggestion
# To avoid having to install these and burst memory limit on ReadTheDocs.
autodoc_mock_imports = [
    "cartopy",
    "clisops",
    "dask",
    "h5py",
    "intake",
    "intake_esm",
    "pandas",
    "rechunker"
    "xarray",
    "xclim",
    "xesmf",
    "zarr",
]
```